docs/investigate_flank_commands.md

```diff
@@ -0,0 +1,121 @@
+# Investigate flank commands
+
+## List of commands android
+
+### gcloud
+
+1. app
+2. test
```

Comment:
In markdown you can use `0.` for all the numbers, and it'll auto number them 🙂 then we don't have to manually renumber in the future.